x11-packages/tigervnc/build.sh

```diff
@@ -2,13 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://tigervnc.org/
 TERMUX_PKG_DESCRIPTION="Suite of VNC servers. Based on the VNC 4 branch of TightVNC."
 TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-# No update anymore. v1.11.x requires support of PAM.
-TERMUX_PKG_VERSION=(1.10.1
+TERMUX_PKG_VERSION=(1.14.1
                     21.1.8)
-TERMUX_PKG_REVISION=41
 TERMUX_PKG_SRCURL=(https://github.com/TigerVNC/tigervnc/archive/v${TERMUX_PKG_VERSION}.tar.gz
```

Comment:
```suggestion
TERMUX_PKG_SRCURL=(https://github.com/TigerVNC/tigervnc/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
```

Reply:
Is there any specific reason to use `refs/tags`?

Reply:
https://github.com/termux/termux-packages/blob/41f130bfed9aab0b89d6565f55af210fdaaba0d5/packages/boxes/build.sh#L28-L30
https://lore.kernel.org/buildroot/87edqhwvd0.fsf@dell.be.48ers.dk/T/

Reply:
Oh, probably we should make linter check this.